### ISSUE TITLE
[BUG] Poor Performance in Examples

### ISSUE THREAD

**Issue by ShaneFlandermeyer** [pytorch/rl#1271]
## Describe the bug

Hello, this is more a question than a bug, but this seemed like a reasonable place to ask. 

It looks like the scripts in the examples subfolder converge to significantly lower rewards than other implementations. For example, the [CleanRL SAC implementation](https://docs.cleanrl.dev/rl-algorithms/sac/) gets a final episode reward of over 10k in HalfCheetah, while the results in EXAMPLES.md show it getting less than 2k for the same number of steps.  Am I misinterpreting the presented results? I've tried the examples using similar hyperparameters as CleanRL and it still does worse, although it converges to ~3k on HalfCheetah.

Similar question to #1181, except I'm going directly from the provided examples. 

## To Reproduce

Running the script directly from the examples directory as

```bash
python3 sac/sac.py
```

## Expected behavior

I would expect a similar reward as in CleanRL, given that everything else is the same.

## System info

```python
import torchrl, numpy, sys
print(torchrl.__version__, numpy.__version__, sys.version, sys.platform)
```

```bash
None 1.24.3 3.10.11 (main, Apr 20 2023, 19:02:41) [GCC 11.2.0] linux
```

## Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)
- [x] I have read the [documentation](https://github.com/pytorch/rl/tree/main/docs/) (**required**)
- [x] I have provided a minimal working example to reproduce the bug (**required**)
